src/Constraint/Node/VariableNode.php

```diff
@@ -39,8 +50,7 @@ public function getName()
      */
     public function evaluate()
```

Comment:
What was this supposed to be anyway?

Reply:
I guess only @tristanlins could answer it.

Maybe accessing properties from some contextual objects (current page, current layout, ...) could make sense.

Reply:
@dmolineus is right, this node was designed to access global variables.